### DISCUSSION TITLE
기대 많이하고 일주일에 두번씩 들여다보고있는데

### DISCUSSION THREAD

**Discussion by goodiii1** [dlunch/wie#346] [Q&A] [answered]
이제 개발은 넘어진건가요? ㅠㅠ

**Comment by dlunch** [comment, marked as answer]
안녕하세요, 기대 가져주셔서 감사합니다.

개발이 중단된 건 아니구요, 저 혼자 작업하는것이다 보니 진행이 빠르게 되지 않을 수 있는 점 양해 부탁드립니다 🙏

**Reply by goodiii1** [reply]
응원합니다~ 사소한 진행사항이라도 적어주세유~
